package.json

```diff
@@ -24,20 +24,20 @@
   "homepage": "https://github.com/wuhanyumsft/type2docfx#readme",
   "devDependencies": {
     "gulp": "^3.9.1",
-    "gulp-tslint": "^8.1.2",
-    "gulp-typescript": "^3.2.2",
-    "tslint": "^5.6.0",
-    "typescript": "^2.4.2"
+    "gulp-tslint": "^8.1.4",
+    "gulp-typescript": "^3.2.4",
+    "tslint": "^5.18.0",
+    "typescript": "^2.9.2"
   },
   "dependencies": {
-    "@types/commander": "^2.9.2",
-    "@types/fs-extra": "^4.0.0",
-    "@types/js-yaml": "^3.9.1",
-    "@types/lodash": "^4.14.74",
-    "commander": "^2.11.0",
+    "@types/commander": "^2.12.2",
+    "@types/fs-extra": "^4.0.9",
+    "@types/js-yaml": "^3.12.1",
+    "@types/lodash": "^4.14.136",
+    "commander": "^2.20.0",
     "fs": "0.0.1-security",
-    "fs-extra": "^4.0.1",
-    "js-yaml": "^3.9.1",
-    "lodash": "^4.17.4"
+    "fs-extra": "^4.0.3",
+    "js-yaml": "^3.13.1",
+    "lodash": "^4.17.15"
```

Comment:
Please keep the dependencies version unchanged if it's not necessary, as it has nothing to do with the target of this PR.